### ISSUE TITLE
Name property for List item in 'Create a new Workbook' window is not defined properly

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#335]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to List item in 'Create new Workbook' and observe Inspect property

### Expected Behavior
Name property should be properly defined for List item  (For e.g. 'WPF')

### Actual Behavior
Name property for list item is defined as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem'  

**Narrator behavior:** Narrator is announcing list item as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem, 4 of 4, selected'
**NVDA behavior**: NVDA is announcing list item as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem,'
**Note:** Same Behavior is also observed for other list items such as 'Console, iOS, Android'
### User Impact
Users who rely on screen reader will not be able to know the Name of List items

![name_property_list_item_inspect](https://user-images.githubusercontent.com/22956130/40721705-ef39881e-6437-11e8-92f0-cf24e3338ba6.png)
![name_property_list_item_narrator](https://user-images.githubusercontent.com/22956130/40721821-3d1f3286-6438-11e8-8f59-30522311bc09.png)
![name_property_list_item_nvda](https://user-images.githubusercontent.com/22956130/40721840-468c72de-6438-11e8-895f-f295b3dd1ed2.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
